api/src/main/java/org/openmrs/module/locationbasedaccess/aop/interceptor/PatientServiceInterceptorAdvice.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * <p>
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.locationbasedaccess.aop.interceptor;
+
+import org.aopalliance.intercept.MethodInterceptor;
+import org.aopalliance.intercept.MethodInvocation;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.Patient;
+import org.openmrs.PersonAttributeType;
+import org.openmrs.User;
+import org.openmrs.api.context.Context;
+import org.openmrs.api.context.Daemon;
+import org.openmrs.module.locationbasedaccess.LocationBasedAccessConstants;
+import org.openmrs.module.locationbasedaccess.aop.PatientSearchAdviser;
+import org.openmrs.module.locationbasedaccess.utils.LocationUtils;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class PatientServiceInterceptorAdvice implements MethodInterceptor {
+
+    private static final Log log = LogFactory.getLog(PatientSearchAdviser.class);
```

Comment:
The log class does not look right.

Reply:
Just fixed it @dkayiwa 
Please have a look again. Thanks